### PR TITLE
3.3 show search result

### PR DESCRIPTION
**Pull request by ingridstake**

### CODE REVIEW

File: project/backend/src/main/java/com/U2/backend/Services/CommandHandlers/SearchService.java [hunk 1]
```diff
@@ -82,6 +82,22 @@ public String performSearch(String searchParam) {
         return null;
     }
 
+
```
Comment (toveenilssonn) [inline project/backend/src/main/java/com/U2/backend/Services/CommandHandlers/SearchService.java]: blankrad

File: project/frontend/src/styles/searchResult.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
+.searchResult {
+
```
Comment (toveenilssonn) [inline project/frontend/src/styles/searchResult.css]: blankrad

File: project/frontend/src/styles/searchResult.css [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+.search-result-name {
+    text-align: left;
+    border-radius: 5px;
+    width: 20rem;
+    padding-left: 10px;
+    font-size: 13pt;
+    text-align: left;
+    vertical-align: middle;  
+    color: #000;
+
```
Comment (toveenilssonn) [inline project/frontend/src/styles/searchResult.css]: blankrad

**Review by toveenilssonn — APPROVED** [review]

**Review by toremark — APPROVED** [review]
Added search service in backend and created support to show search results in frontend.

**Pull request merged**